### DISCUSSION TITLE
Where to report error in CVE writeup?

### DISCUSSION THREAD

**Discussion by ColBFritz** [curl/curl#10864] [Q&A] [answered]
The curl project maintains excellent [documentation of security issues](https://curl.se/docs/security.html) found in the software including technical analysis of the problem, patch information, affected versions, and recommendations for mitigation in the case a patch cannot be applied.

We have recently found an error in the writeup for a CVE on this page meaning the suggested mitigation does not actually protect against the issue. This likely leaves unpatched systems vulnerable even if they have attempted to apply the workaround.

Where should I report this so that the CVE guidance can be corrected in the curl security documentation? This is not a new issue so I don't want to open a new hackerone report, but I could not find any other security-related contact information. I wanted to be sure the security team had an opportunity to verify the issue and potentially find a new mitigation to recommend before I reported it publicly.

**Comment by dfandrich** [comment]
You can reach the security team privately at ***@***.***

**Reply by ColBFritz** [reply]
Thanks for the response! Unfortunately GitHub appears to have sanitized the email address and all I see is asterisks. Is this the "security (at curl dot se)" private email list which is noted on https://curl.se/dev/secprocess.html ? Or some other address? I had the impression from reading that page, the private list required membership to post.

**Comment by dfandrich** [comment, marked as answer]
Yes, that's it. We should make it more clear that it's fine for anyone to send mail there to discuss a security issue.

**Reply by ColBFritz** [reply]
Thank you, I have sent a message to that mailing list address (security "at" curl "dot" se) detailing my finding.
